storage.go

```diff
@@ -490,7 +501,7 @@ func (s Storage) GetLeaderHostname(ctx context.Context) (string, error) {
 	}
 
 	if len(resp.Kvs) == 0 {
-		return "", errors.New("no leader")
```

Comment:
Why do you change this line?
`ErrLeaderNotExist` is used only here.  It seems not worthy of a constant.

Reply:
`ErrLeaderNotExist` was once used in multiple code portions, which have been removed now.  Reverted.